packages/client/mutations/AddReactjiToReactableMutation.ts

```diff
@@ -104,7 +113,19 @@ const AddReactjiToReactableMutation: StandardMutation<TAddReactjiToReactableMuta
         }
       }
     },
-    onCompleted,
+    onCompleted: (res, errors) => {
+      const {isRemove} = variables
+      const addedKudos = res.addReactjiToReactable.addedKudos
+      if (!isRemove && addedKudos) {
+        const {unicode} = getReactji(addedKudos.emoji)
+        atmosphere.eventEmitter.emit('addSnackbar', {
+          key: 'youGaveKudos',
+          message: `You gave kudos to ${addedKudos.receiverUser.preferredName} ${unicode}`,
```

Comment:
The kudos snackbar is showing up a bit right of center for me

![kudos](https://github.com/ParabolInc/parabol/assets/39854876/9f182f3b-4d30-4bb0-91a7-f957f5ac440c)

Reply:
It is currently same for all snackbars, it is centered using right side content only, not including sidebar. Not sure if we need to change it, but I can create a separate issue in this case

<img width="1735" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/dc184e7b-3c91-4d10-be70-27e8255ef37b">